packages/graph/hash_graph/lib/graph/src/shared/identifier/time/mod.rs

```diff
@@ -0,0 +1,32 @@
+mod axis;
+mod timestamp;
+mod version;
+use std::{error::Error, ops::Bound};
+
+use postgres_types::{FromSql, Type};
+
+pub use self::{
+    axis::{
+        DecisionTime, DecisionTimeVersionTimespan, DecisionTimestamp, TransactionTime,
+        TransactionTimeVersionTimespan, TransactionTimestamp,
+    },
+    timestamp::Timestamp,
+};
+
+fn parse_bound(
+    bound: &postgres_protocol::types::RangeBound<Option<&[u8]>>,
+) -> Result<Bound<Timestamp<()>>, Box<dyn Error + Send + Sync>> {
+    match bound {
+        postgres_protocol::types::RangeBound::Inclusive(Some(bytes)) => Ok(Bound::Included(
```

Comment:
Do we want to match the naming by the way? As in `Bound::Inclusive`

Reply:
We neither define any of these variants. `Inclusive` is from `postgres_types`, while `Included` is from `std::ops`.